relay-config/src/config.rs

```diff
@@ -576,6 +576,22 @@ impl Default for Limits {
     }
 }
 
+/// Controls traffic steering.
+#[derive(Debug, Default, Deserialize, Serialize)]
+#[serde(default)]
+pub struct Routing {
+    /// Accept and forward unknown Envelope items to the upstream.
+    ///
+    /// Forwarding unknown items should be enabled in most cases to allow proxying traffic for newer
+    /// SDK versions. The upstream in Sentry makes the final decision on which items are valid. If
+    /// this is disabled, just the unknown items are removed from Envelopes, and the rest is
+    /// processed as usual.
+    ///
+    /// Defaults to `true` for all Relay modes other than processing mode. In processing mode, this
+    /// is disabled by default since the item cannot be handled.
+    unknown_items: Option<bool>,
```

Comment:
I thought the same as joris but I get the reasoning from jan as well, so i don't have an opinion on this

Reply:
I thought it would make sense to make it consistent with the function name, but I did not consider the `routing.` prefix. With the prefix I think it's clear enough.